packages/tests/server/adapters/fetch.test.ts

```diff
@@ -244,6 +256,26 @@ describe('with default server', () => {
 
     await client.foo.query();
   });
+
+  // Miniflare seems broken. Working test in examples/bun
```

Comment:
Could you elaborate on this comment a bit?

Reply:
Test failed when it shouldn't